docker/entrypoint.sh

```diff
@@ -15,4 +15,9 @@ if [[ -z "${DEPLOY_ENV}" ]]; then
     ./codecov -t -t ${CODECOV_TOKEN} -F "${DEVITO_ARCH}-${DEVITO-PLATFORM}"
 else
     exec "$@"
-fi
\ No newline at end of file
+fi
+
+if [[ -n "${ROCM_VERSION}" ]]; then
+    echo "sourcing Omnitrace"
+    source /opt/omnitrace/share/omnitrace/setup-env.sh
```

Comment:
may need a `COPY chown /opt` (pseudo code) for safety

Reply:
I don't think so those files are readable

Reply:
Since you are using modulefiles for omniperf, might as well use `module use /opt/omnitrace/share/modulefiles`

Reply:
thanks, will do that